README.md

```diff
@@ -286,6 +286,7 @@ The following are only in the resource subject (that is, the base subject).
   label: <label for uri>,
   valueSubjectKey: <key for subject for a nested resource>,
   -> valueSubject: {<subject>}
+  -> index: <1 based index of the value (relative to siblings)>
```

Comment:
I thought `->` means not stored in state, but this is putting it in state, isn't it?

Reply:
No, it is being injected by the selector.